lib/acts_as_list/active_record/acts/position_column_method_definer.rb

```diff
@@ -21,11 +23,11 @@ def self.define_class_methods(caller_class, position_column)
       end
 
       define_singleton_method :decrement_all do
-        update_all_with_touch "#{quoted_position_column} = (#{quoted_position_column_with_table_name} - 1)"
+        update_all_with_touch "#{quoted_position_column} = (#{quoted_position_column_with_table_name} - 1)".dup
```

Comment:
Sorry, I'm feeling dense today :) Can you explain why we need to dup the string? :)

Reply:
[Here](https://github.com/swanandp/acts_as_list/blob/master/lib/acts_as_list/active_record/acts/position_column_method_definer.rb#L32) this string is concatenated but one cannot change a frozen string. :)

Reply:
Could we instead unfreeze (`.dup`?) `updates` in:

```
update_all(updates.dup << touch_record_sql)
```

?

Reply:
In this case, we won't mutate `updates` so maybe we can change it to concatenation?
```ruby
update_all(updates + touch_record_sql)
```

Reply:
Yep, that't cleaner :)

Reply:
@brendon changed! :)

Reply:
Thanks @krzysiek1507, I'll just wait on @swanandp's check and then we'll be GTG.